extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -62,6 +62,23 @@ export const buildOpenInlineAttachmentElement = (element) =>
       </div>
     </a>`;
 
+/**
+ * @param {!Element} element
+ * @return {!Element}
+ */
+ const buildOpenOutlinkAttachmentElement = (element) =>
+ htmlFor(element)`
+     <a class="i-amphtml-story-page-open-attachment"
+         role="button">
+       <span class="i-amphtml-story-outlink-page-attachment-arrow">
+         <span class="i-amphtml-story-outlink-page-open-attachment-bar-left"></span>
```

Comment:
I wonder if we could use the same class names for this and the page attachment, since they have the same UI.

Reply:
They have slightly diff styling, but they will have different animation soon so I think it's better to keep as separate classes

Reply:
Is the animation / styling difference for the `bar-left/right` spans or the component itself? I meant this comment just for the arrow icon creation, which seems to be identical. If the arrow icon is different, then no need to change anything

Reply:
ah, it was called `i-amphtml-story-page-open-attachment-icon` before and you mean using that again instead of `i-amphtml-story-outlink-page-attachment-arrow`

I think it wouldn't work bc there is also a `.i-amphtml-story-page-open-attachment-icon::after` and we don't want that to be added in this case